src/stapi_fastapi/models/shared.py

```diff
@@ -9,6 +9,8 @@ class Link(BaseModel):
     type: Optional[str] = None
     title: Optional[str] = None
     method: Optional[str] = None
+    headers: Optional[dict[str, Union[str, list[str]]]] = None
+    body: Any = None
```

Comment:
Would this be better typed as `Optional[dict[str, Any]]` to represent that it must be null or a json object? Or is that too strict?

Reply:
I left it as any since that's what the spec had...I actually had it as your suggestion then reverted it. Figure we can narrow later?